run_tests.py

```diff
@@ -482,3 +488,26 @@ def test_stdin(self, stdin_get_value):
             len(ret),
             1,
         )
+
+    @pytest.mark.skipif(
+        sys.version_info < (3, 5),
+        reason='This syntax is only valid in Python 3.x',
+    )
+    def test_tuple_unpacking(self):
+        tree = ast.parse('a, *(b, c) = 1, 2, 3')
+        checker = BuiltinsChecker(tree, '/home/script.py')
+        ret = [c for c in checker.run()]
+        self.assertEqual(len(ret), 0)
+
+
+if sys.version_info >= (3, 6):
+    @given(source_code=hypothesmith.from_grammar())
+    def test_builtin_works_on_many_examples(source_code):
+        try:
+            source = source_code.encode('utf-8-sig')
+        except UnicodeEncodeError:
+            reject()
+            raise
+        tree = ast.parse(source)
+        checker = BuiltinsChecker(tree, '/home/script.py')
+        assert isinstance([c for c in checker.run()], list)
```

Comment:
This assertion can never fail, because the list comprehension is always a list.  Maybe you meant:

```python
for c in checker.run():
    assert isinstance(c, list)
```

Reply:
Good point!

Fixed it on https://github.com/gforcada/flake8-builtins/commit/fac8ea78def6cc2777fa6e27b26b374046621a40